src/topology/continuous_function/compact.lean

```diff
@@ -394,4 +423,28 @@ end
 
 end comp_right
 
+section weierstrass
+variables {X : Type*} [topological_space X] [t2_space X] [locally_compact_space X]
+variables {E : Type*} [normed_group E] [complete_space E]
+
+/-- **Weierstrass M-test** -/
+lemma summable_of_locally_summable_norm {ι : Type*} {F : ι → C(X, E)}
+  (hF : ∀ K : set X, compact_space K → by exactI summable (λ i, ∥(F i).restrict K∥)) :
+  summable F :=
+begin
+  classical,
```

Comment:
There's already an `open_locale classical` at the top of the file (which is apparently unused until now, though)

```suggestion
```

Reply:
Actually I think we should remove `open_locale classical` from this file and restore the `classical` here.

It doesn't matter here at the moment because no `def`s in the file need a decidability hypothesis but if that ever changes then `open_locale classical` potentially becomes a source of pain through [decidability diamonds](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/A.20possible.20diamond/near/260159581).

Reply:
In fact while you're changing the `open_locale` line I suggest adding `uniformity` instead to get the `𝓤` notation.